src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx

```diff
@@ -48,7 +48,9 @@ export const PopularSpeciesButton = (props: Props) => {
 
   const handleClick = () => {
     const { genome_id, is_available } = species;
-    const speciesName = species.common_name || species.scientific_name;
+    const speciesName = `${species.common_name || species.scientific_name} - ${
+      species.assembly_name
+    }`;
```

Comment:
It looks like we are now doing the same calculation in 4 different places? Perhaps extract into a helper (`getSpeciesAnalyticsName`)?